### PR TITLE
Document discovery_splash and features in editGuild

### PR DESCRIPTION
**Pull request by Khaaz**
discord/discord-api-docs#1764

### CODE REVIEW

**Comment by bsian03** [comment]
TS docs needed

**Comment by cyclane** [comment]
You should also add `discoverySplash`, `discoverySplashURL` and `dynamicDiscoverySplashURL` to Guild and `discoverySplashURL` to GuildPreview.

**Comment by bsian03** [comment]
ignore above comment, i didn't realise it was for something different

File: index.d.ts [hunk 1]
```diff
@@ -612,6 +612,8 @@ declare namespace Eris {
     systemChannelFlags: 0 | 1 | 2;
     systemChannelID?: string;
     verificationLevel?: number;
+    discoverySplash: string;
+    features: string[];
```
Comment (bsian03) [inline index.d.ts]: Both should be alphabetically positioned and both should also be optional
```suggestion
    discoverySplash?: string;
    features?: string[];
```

File: index.d.ts [hunk 2]
```diff
@@ -612,6 +612,8 @@ declare namespace Eris {
     systemChannelFlags: 0 | 1 | 2;
```
Comment (bsian03) [inline index.d.ts]: Not sure why this isn't optional but may as well fix it here

**Review by bsian03 — CHANGES_REQUESTED** [review]

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**